install/uninstall.ps1

```diff
@@ -0,0 +1,125 @@
+# Enable a default -Verbose flag for debug output.
+[CmdletBinding()]
+
+# Halt script execution at the first failed command.
+$script:ErrorActionPreference='Stop'
+
+$azureauthDefaultLocation = ([System.IO.Path]::Combine($Env:LOCALAPPDATA, "Programs", "AzureAuth"))
+
+if (![string]::IsNullOrEmpty($Env:AZUREAUTH_INSTALL_DIRECTORY) `
+        -And $Env:AZUREAUTH_INSTALL_DIRECTORY -ne $azureauthDefaultLocation) {
+    Write-Warning "Ignoring AZUREAUTH_INSTALL_DIRECTORY environment variable. Custom uninstallation locations are not supported."
+}
+
+function Uninstall {
+    Close-AzureauthInstances
+
+    $locations = Get-AzureauthsInPath
+    Remove-AzureauthDirectories $locations
+    Remove-FromPath $locations
+
+    Write-Output "Uninstalled AzureAuth!"
+}
+
+function Get-AzureauthsInPath {
+    $allLocations = [System.Collections.ArrayList]@()

```

Comment:
Unless I'm mistaken this doesn't get used for **all** locations, only for those which are subdirectories of the default installation directory. Other directories are iterated over, but ultimately ignored.

Consider renaming this variable accordingly.

Reply:
Changed the name.